### ISSUE TITLE
Create 3 support

### ISSUE THREAD

**Issue by jd1900** [MrSpike63/vanity-eth-address#2]
It would be great to add create3 support for just using the salt and not needing the initcode hash.

**Comment by MrSpike63** [comment]
As far as I can tell, https://github.com/ethereum/EIPs/pull/3171 isn't official, but I have added support for [create3-factory](https://github.com/ZeframLou/create3-factory). It takes the address of the caller, deployer contract, and the bytecode of the deployer's proxy contract, as there are multiple deployer contracts to choose from.

**Comment by NaliFinance** [comment]
i may doing it wrong, but given the contract bytecode, the deployer-address (0x9fBB3DF7C40Da2e5A0dE984fFE2CCB7C47cd0ABf) and the msg.sender (my) address.. i get a salt, but when i test it 
https://sepolia.etherscan.io/address/0x9fBB3DF7C40Da2e5A0dE984fFE2CCB7C47cd0ABf#readContract

it gives me a different final contract address.. what am i missing?

**Comment by MrSpike63** [comment]
Just to confirm, did you set the bytecode to `0x67363d3d37363d34f03d5260086018f3`?

**Comment by NaliFinance** [comment]
> Just to confirm, did you set the bytecode to `0x67363d3d37363d34f03d5260086018f3`?

nope, but it's working now! where i could find that? i used the bytecode from etherscan and it didn't work, with this it's working! great!!

**Comment by MrSpike63** [comment]
The bytecode needed is the bytecode of the *proxy* contract the deployer uses, not the deployer itself. You can find it [here](https://sepolia.etherscan.io/address/0x9fBB3DF7C40Da2e5A0dE984fFE2CCB7C47cd0ABf#code#F2#L33).

**Comment by NaliFinance** [comment]
damn, thank you! i didn't understand that at first glance.. thank you again for all the help! +++
